miqcli/collections/automation_requests.py

```diff
@@ -145,6 +145,8 @@ def status(self, req_id):
             log.info(' * ID: %s' % req_id)
             for key, value in status.items():
                 log.info(' * %s: %s' % (key.upper(), value))
+            # if verbosity is set, get more info about the request
+            log.debug(req.options)
```

Comment:
Would it be possible to better format this data returned to the user? Currently the user will get something like the following:

`
DEBUG: {u'user_id': 1, u'class_name': u'Methods', u'namespace': u'CF_MIQ_CLI/General', u'delivered_on': u'2018-03-12T12:27:22.667Z', u'instance_name': u'get_floating_ip', u'attrs': {u'cloud_network_id': 2, u'cloud_tenant_id': 1, u'userid': u'admin'}, u'return': u'{"status":"success","return":{"<ip_address>":<ip_address_id>}}'}
`

If we used the [pformat](https://docs.python.org/3/library/pprint.html#pprint.pformat) from the pprint module, we could achieve something similar to below. It is just easier for reading purposes.

```python
from pprint import pformat

log.debug(pformat(req.options, indent=2))
```
```python
{ 'attrs': {'cloud_network_id': 2, 'cloud_tenant_id': 1, 'userid': 'admin'},
  'class_name': 'Methods',
  'delivered_on': '2018-03-12T12:27:22.667Z',
  'instance_name': 'get_floating_ip',
  'namespace': 'CF_MIQ_CLI/General',
  'return': '{"status":"success","return":{"<ip_address>":<ip_address_id>}}',
  'user_id': 1}
```